### DISCUSSION TITLE
Chem Controller Mixing Time for all chems

### DISCUSSION THREAD

**Discussion by camaro4life18** [tagyoureit/nodejs-poolController#930] [Nixie]
Hello, I started using Nixie a while back for my pool, and have has some pretty good success with it. This winter I added my Hot Tub as a 2nd body. I've found managing the chemicals is much more sensitive given how its only 335 gallons of water. My problem is I have set both the Acid and Chlorine dosing to have 15 minute timers, but the problem is when I dose Chlorine, it immediately notices the increase in pH and starts dosing Acid. 
I've set my pH to 7.6 in the hopes it won't dose as much, but the pH will spike above 8 while its mixing. So I end up with an average pH of 7.0 once its all settled. How can I prevent the the Acid dosing while the Chlorine is mixing?

**Comment by tagyoureit** [comment]
Try dosing ORP by volume and setting it to a low value so it doesn't spike the pH.  15 minutes of dosing could be quite a bit for the smaller size hot tub.
